include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf_pressure_correction.cpp

```diff
@@ -357,6 +357,24 @@ TimeIntBDFPressureCorrection<dim, Number>::momentum_step()
       velocity_np = velocity_momentum_last_iter;
     }
 
+    /*
```

Comment:
Good, the nonlinear path with implicit convection and extrapolated viscous term now also has a viscosity update! Good that you caught that, this was actually wrong in old code versions also prior to #671 !!